src/input.rs

```diff
@@ -324,12 +322,15 @@ pub struct Stdin<T: Into<String>>(pub T);
 /// assert_eq!(output, "bar\nfoo\n");
 /// # }
 /// ```
+///
+/// If `Stdin` is used multiple times, all given bytes slices will be written
+/// to the child's standard input in order.
```

Comment:
I moved this sentence below the code example. I think that makes more sense. WDYT, @casey?

Reply:
Looks good to me!